crates/nu-cmd-extra/src/extra/bits/or.rs

```diff
@@ -42,14 +59,37 @@ impl Command for BitsOr {
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let head = call.head;
-        let target: i64 = call.req(engine_state, stack, 0)?;
+        let target: Value = call.req(engine_state, stack, 0)?;
+        let endian = call.get_flag::<Value>(engine_state, stack, "endian")?;
+
+        let little_endian = match endian {
+            Some(val) => {
+                let span = val.span();
+                match val {
+                    Value::String { val, .. } => match val.as_str() {
+                        "native" => cfg!(target_endian = "little"),
+                        "little" => true,
+                        "big" => false,
+                        _ => {
+                            return Err(ShellError::TypeMismatch {
+                                err_message: "Endian must be one of native, little, big"
+                                    .to_string(),
+                                span,
+                            })
+                        }
+                    },
+                    _ => false,
+                }
+            }
+            None => cfg!(target_endian = "little"),
+        };
 
-        // This doesn't match explicit nulls
```

Comment:
This comment too (sorry, I'd make a suggestion to make it easier for you to just click the button and commit but GitHub doesn't let you do that on deleted lines)